src/js/components/Clock/Digital.js

```diff
@@ -38,20 +43,38 @@ const Digit = ({ number, run, size }) => {
     );
   }
   return (
-    <StyledDigitalDigit size={size}>{Math.floor(number)}</StyledDigitalDigit>
+    <StyledDigitalDigit
+      size={size}
+      {...(withinThemeContext === undefined ? { theme } : {})}
+    >
+      {Math.floor(number)}
+    </StyledDigitalDigit>
   );
 };
 
 const Element = ({ number, run, sep, size }) => {
+  const theme = useThemeValue();
+  const withinThemeContext = useContext(ThemeContext);
   const tens = Math.floor(number / 10);
   const ones = number % 10;
   const result = [
-    <Digit key="tens" run={run} size={size} number={tens} />,
+    <Digit
+      key="tens"
+      run={run}
+      size={size}
+      number={tens}
+      theme={theme}
```

Comment:
Soft opinion but this feels a little like prop drilling. Should we just have lines 56/57 in Digit as well as opposed to passing down as props here?

Fine either way, just stuck out to me.

Reply:
fixed in latest commit